backend/app/views/main.py

```diff
@@ -139,3 +139,16 @@ def scrap_all_schedule():
         password=password
     )
     return jsonify(response), status_code
+
+@router_main.route('/courses', methods=['GET'])
+def get_all_courses():
```

Comment:
@angga1518 Can you give the response preview of this endpoint in the pull request description?

Reply:
updated